### ISSUE TITLE
Sort order by custom prefix

### ISSUE THREAD

**Issue by srcspider** [MLoughry/sort-typescript-imports#20]
I use custom `NODE_PATH` set to the project so as to avoid `../../../..` hell in imports. While I don't use it I also know of people who use a `@project_folder` format of doing similar thing (I've seen a few other methods, such as ones using webpack, etc).

_The main commonality is that you have a bunch of root names that are actually project imports and not npm imports even if they might not look like it._

My suggestion is to add the following two options to `pathSortOrder`:
 - `subpackages` ie. anything that has a `/` in the path but doesn't start with a `./` or `../` or `/`; this almost automatically detects most of the patterns mentioned earlier, but might still get false positives ocasionally on modules like `lodash/each` (they're rare in my experience)
 - `customPathPrefixes` which matches against a new option `sortImports.customPrefixOrder`

Example of `sortImports.customPrefixOrder` 

```ts
"typescript.extension.sortImports.customPrefixOrder": [
   "core/lib",
   "core",
   "common",
   "common/server"
]
```
The list would be a bit longer but the above covers the corner cases IMO. And while not that short, there's usually not many of these prefixes.

---

ps. for completion sake you might also want to have...

 - `absolutePath` ie. anything that starts with `/` (this can happen in Docker instances where it's all very compact and absolute paths are practical -- not necessarily in regular source files)

**Comment by MLoughry** [comment]
With the release of [Typescript 2.8](https://devblogs.microsoft.com/typescript/announcing-typescript-2-8-2/#organize-imports) over a year ago, the ability to organize imports comes bundled with Typescript, in a better implementation than this one made for a hack week. As such, I have no further plans to maintain this project.

If for whatever reason you would like to take ownership of this repository, please let me know.
